src/Core/Framework/Adapter/Cache/CacheStateSubscriber.php

```diff
@@ -51,10 +51,10 @@ public function login(CustomerLoginEvent $event): void
 
     public function cartChanged(CartChangedEvent $event): void
     {
-        $event->getContext()->removeState(self::STATE_CART_FILLED);
+        $event->getSalesChannelContext()->removeState(self::STATE_CART_FILLED);
```

Comment:
Remember to change this back to `getContext()`

Reply:
Why? We're adding this method here and deprecating the incorrectly typed getContext method, why not already changed dependents on the deprecated method?

Reply:
Ah you are correct. My apologies. I typed this, then went back to my previous comment and changed it. 😅